community/modules/scheduler/gke-cluster/README.md

```diff
@@ -1,5 +1,59 @@
+## Description
+
+This module creates a Kubernetes cluster using
```

Comment:
nit: wondering if you should just say: this module creates a [GKE](https://cloud.google.com/kubernetes-engine) cluster

Reply:
I wanted to include `Kubernetes` somewhere in case people search for it (I believe this was your idea). I could instead say `This module creates a Google Kubernetes Engine (GKE) cluster.`

Reply:
I have updated to use the `This module creates a Google Kubernetes Engine (GKE) cluster.` wording.